### PR TITLE
Chat SDK reconnect tests

### PR DESCRIPTION
**Pull request by gitanjuvs**
This PR includes the following authenticated scenario,

1. ChatSDK.getChatReconnectContext() should return a reconnect id if there's an existing chat session.

### CODE REVIEW

File: playwright/integrations/authenticated-chat-with-reconnect.spec.ts [hunk 1]
```diff
@@ -67,4 +67,89 @@ test.describe('AuthenticatedChat @AuthenticatedChatWithChatReconnect', () => {
         expect(reconnectableChatsRespense.status()).toBe(204);
         expect(reconnectableChatsRequestHeaders['authenticatedusertoken']).toBe(authToken);
     });
+
+    test('ChatSDK.getChatReconnectContext() should return a reconnect id if theres an existing chat session', async ({ page }) => {
+        await page.goto(testPage);
+
+        const [_, reconnectableChatsRequest, reconnectableChatsRespense, runtimeContext] = await Promise.all([
```
Comment (xTEddie) [inline playwright/integrations/authenticated-chat-with-reconnect.spec.ts:74]: `reconnectableChatsRespense` -> `reconnectableChatsResponse`

File: playwright/integrations/authenticated-chat-with-reconnect.spec.ts [hunk 2]
```diff
@@ -67,4 +67,89 @@ test.describe('AuthenticatedChat @AuthenticatedChatWithChatReconnect', () => {
         expect(reconnectableChatsRespense.status()).toBe(204);
         expect(reconnectableChatsRequestHeaders['authenticatedusertoken']).toBe(authToken);
     });
+
+    test('ChatSDK.getChatReconnectContext() should return a reconnect id if theres an existing chat session', async ({ page }) => {
```
Comment (xTEddie) [inline playwright/integrations/authenticated-chat-with-reconnect.spec.ts:71]: `theres` -> `there's`

Comment (gitanjuvs) [reply playwright/integrations/authenticated-chat-with-reconnect.spec.ts]: Changed.

File: playwright/integrations/authenticated-chat-with-reconnect.spec.ts (hunk 1 above)
Comment (gitanjuvs) [reply playwright/integrations/authenticated-chat-with-reconnect.spec.ts]: Changed

**Review by xTEddie — APPROVED** [review]

**Pull request merged**